src/App.tsx

```diff
@@ -14,49 +15,109 @@ export const goodsFromServer = [
   'Jam',
   'Garlic',
 ];
+enum SortType {
+  CONST_STATE_BY_LENGTH = 'byLength',
+  CONST_STATE_BY_ALPHABET = 'byAlphabet',
+}
```

Comment:
enum's elements should not contain const prefix
```suggestion
enum SortType {
  STATE_BY_LENGTH = 'byLength',
  STATE_BY_ALPHABET = 'byAlphabet',
}
```